.stitch/designs/chat_desktop.html

```
@@ -0,0 +1,326 @@
+<a class="text-[#5e5680] dark:text-slate-400 hover:text-indigo-500 transition-colors" href="#">Profile</a>
+</div>
+</div>
+<div class="flex items-center gap-4">
+<button class="p-2 hover:bg-[#f4eeff] dark:hover:bg-slate-800 rounded-lg transition-all duration-200">
+<span class="material-symbols-outlined text-on-surface-variant">notifications</span>
+</button>
+<button class="p-2 hover:bg-[#f4eeff] dark:hover:bg-slate-800 rounded-lg transition-all duration-200">
+<span class="material-symbols-outlined text-on-surface-variant">settings</span>
+</button>
+<div class="w-10 h-10 rounded-full bg-primary-container overflow-hidden">
+<img class="w-full h-full object-cover" data-alt="Close-up portrait of a creative professional with modern glasses in a bright studio setting with soft natural light" src="https://lh3.googleusercontent.com/aida-public/AB6AXuD9NqqrwmEdpKPmSJOdWAAD_n36dVIP8oXscGl_TzhgIyFlt35JoC2ftAgRMQIDc5Jr-l4rltbUN-weoswJWRJX5UQ76G6eim7_xzZI0eaTuQN64rU9w0-AA9fJiuwJGZTb13mT_UdgT7wG2gGiN3wU05PIql9zVB6L0beFukn9pDrmo0B0WVifU9y2yzGNds6gQYOWIvT9JQ3SGlQPVcI7WEdFaq-nL4swEfxyksVfeBOxchPAotk3w0DI0JGNF_X8PmGZvxc4-4g"/>
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)